exercises/binary-search-tree/src/test/java/BinarySearchTreeTest.java

```diff
@@ -6,15 +6,21 @@
 import static org.junit.Assert.assertNotNull;
 import org.junit.Ignore;
 import org.junit.Test;
+import org.junit.Before;
 
-public class BSTTest {
+public class BinarySearchTreeTest {
+    BinarySearchTree<Integer> bst;
```

Comment:
Just as a heads up, this will probably be changed as part of  issue #230 so that not all the binary search trees used in the tests will be of type `Integer`. In which case we won't be able to save it as a private variable anymore :( So it's fine to leave it as it is for now, as the tests are it's a really good change :) But just be aware that it might be changed back in the near future :)

Also also, if you do decide to keep it then it should be made `private` :)